BREAKING.md

```diff
@@ -33,6 +34,9 @@ In a subsequent major release, the policy `IDocumentStorageServicePolicies.maxim
 (and likewise `IDocumentStorageService.policies` itself) may become required,
 to ensure all drivers take note of this requirement and enforce this policy.
 
+### Added `fetchTokenError` errorType to DriverErrorType enum
+Added `fetchTokenError` errorType in DriverErrorType enum. This error happens when a token providers `fetchOrdererToken` call fails.
```

Comment:
I'd probably not add anything, or add something here saying that no changes are required by consumers. Otherwise, it's not very clear for readers what they need to do to react to this record.